src/components/select/index.tsx

```diff
@@ -8,6 +8,8 @@ import ChipsComponent from './chips-component';
 import SelectComponent from './select-component';
 
 export interface SelectInputProps extends SelectProps, InputComponent {
```

Comment:
we are using SelectInputProps and ComponentProps in the './select-components.tsx' file, these interfaces are almost the same and extend the same interfaces (SeleProps, InputComponent), let's rewrite it a bit and extend types from there

```
import type { ComponentProps } from './select-component';

export interface SelectInputProps extends ComponentProps {
  clearChips?: boolean;
  hideChips?: boolean;
}
```